docker/servicesmw/Dockerfile-nginx

```diff
@@ -0,0 +1,23 @@
+FROM nginx:1.15.6-alpine
+
+RUN chown -R nginx:nginx /etc/nginx && \
+    chown nginx:nginx /var/cache/nginx && \
+    chown -R nginx:nginx /var/log/nginx && \
+    mkdir /var/run/nginx && \
+    chown nginx:nginx /var/run/nginx
+
+ADD docker/base/base.inc /etc/nginx/conf.d/base.inc
+ADD docker/base/nginx.conf /etc/nginx/nginx.conf
+ADD docker/sandbox/site.conf /etc/nginx/conf.d/default.conf
```

Comment:
You're adding `site.conf` from the `sandbox` dir and I guess everything works. I'm wondering if is would not be better to reuse some shared nginx config instead of copying it in each env. What do you think?